apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -163,3 +203,10 @@ const ScriptEditor = React.createClass({
 });
 
 export default ScriptEditor;
+
+function make_selection_handler(flag) {
+  return function (e) {
+    e.preventDefault();
+    $(this).parent().siblings('select').children('option')[flag ? 'attr' : 'removeAttr']('selected', true);
+  };
+}
```

Comment:
This is only used once (in `componentDidMount`). Why make it a global function rather than inline anonymous function where it's used? Do you intend to reuse it later? Should it at least be a member of the `ScriptEditor` React class?

Though it may not be needed at all (see above comment).